### DISCUSSION TITLE
How can i get the column name or row number when we upload attachment

### DISCUSSION THREAD

**Discussion by ramsigmoid** [nocodb/nocodb#6031] [Q&A]
![image](https://github.com/nocodb/nocodb/assets/98207372/832fe922-b648-4c66-b7fa-65f91b1f2cee)

I want to upload different type of images to different folder, so i just want to have row which has been click or column name which has been clicked.

code which i am using :- "\nocodb\packages\nc-gui\components\cell\attachment\utils.ts"

**Comment by o1lab** [comment]
Upload attachement API is an independent API. You will have to use the return value of this API to update the attachment column of your table columnm

**Reply by ramsigmoid** [reply]
ok, but can i get row number in which upload attachment has been click.

**Reply by o1lab** [reply]
enable system fields within `fields` menu to see the id/

**Reply by ramsigmoid** [reply]
ok, but i need to access programmatically in code. Below function just file, file name , size etc.. But no information which row has been click or which column has been click, can i pass this value from another form?
![image](https://github.com/nocodb/nocodb/assets/98207372/9b972c4a-f438-4ab6-b9f2-a2a86e5e0990)

**Reply by o1lab** [reply]
Use [list api](https://apis.nocodb.com) of the table. If you mean which row or column is "click" on UI - its not possible.
